### ISSUE TITLE
License - Asking for Special Permission (Dropping No-Derivs Clause) for Ruby for Update on How I Start by Steve Klabnik

### ISSUE THREAD

**Issue by geraldb** [howistart/howistart#51]
Hello, 
  first thanks for this great initiative and article series. I'm a ruby guy and have published the How I Start article by Steve Klabnik in the Yuki & Moto Press Bookshelf.

Since the article was published in 2014 some things might need a little update in 2018 - I asked the author Steve Klabnik to drop the no-derivs clause from the creative commons license for this article.  And Steve Klabnik is ok if you are ok and give permission.

Thus, may I ask you to give special permission to drop the no-derivs clause from the creative commons license?

Thanks for your considerations. Happy new year. Prosit 2018. Greetings from Vienna.

**(Update)** - Looking again at the [/howistart.org repo](https://github.com/howistart/howistart.org/) I see that it's all GNU GPL - so for now I also add this license and assume it covers the included articles.

**Comment by tsloughter** [comment]
@geraldb would you be interested in also sending a PR to https://github.com/howistart/howistart-hakyll instead of having them diverge?

**Comment by geraldb** [comment]
@tsloughter   I made small edits (not really any updates) e.g. to keep it simple I changed `$ bundle exec rake` to `$ rake`  (eg removing `bundle exec` in calling commands - that's just a personal preference really) and I removed all `$ cat lib/how_i_start.rb lines` so every code block is a (standard) ruby file.     And I added an appendix with all the source code (see <https://github.com/yukimotopress/start/blob/master/sources.md>).  To sum up nothing changed really (so far).  Again thanks for the great How I Start initiative - love the article series (and high-quality) - fantastic articles (and in-depth). Keep it up. Cheers.
